subworkflows/local/bam_variant_calling_gridss/main.nf

```diff
@@ -2,15 +2,16 @@
 // Run Gridss
 //
 
-include { GRIDSS_GRIDSS      } from '../../../modules/nf-core/gridss/gridss/main'
+include { SIMPLE_EVENT_ANNOTATION   } from '../../../modules/local/gridss/simple_event_annotation/main'
 
-include { TABIX_TABIX        } from '../../../modules/nf-core/tabix/tabix/main'
+include { GRIDSS_GRIDSS             } from '../../../modules/nf-core/gridss/gridss/main'
+include { TABIX_TABIX               } from '../../../modules/nf-core/tabix/tabix/main'
 
 workflow BAM_VARIANT_CALLING_GRIDSS {
     take:
         crams                   // channel: [mandatory] [ meta, cram, crai ] => The aligned CRAMs per sample with the regions they should be called on
```

Comment:
why cram and not ch_cram like for manta and delly?

Reply:
The gridss and whamg subworkflows are a WIP and aren't used currently. (Basically me being a bit lazy and only updating the code that's actually being used at the moment 😁)